.github/CONTRIBUTOR_GUIDELINES.md

```diff
@@ -92,6 +93,53 @@ Aside from the actual changes to the repo that you make, opening a PR has some r
 * Your suggested changelog can be changed by devs prior to being merged if the merging developer is dissatisfied with it. This will be done by editing the initial comment on the PR made by the contributor so our bot can parse that text and add it to the viewable in-game changelog.
 * If you use sprites from another user, do your best to credit them in a changelog bullet entry.
 
+## Reviewing PRs and Issues
+As you might be able to tell, there are a lot of pull requests pending at any given moment in time. In order for a pull request to get merged, developers have to review and approve of it, but contributors can also review pull requests, as well as [add labels where appropriate](#Triage-Team).
+
+### Labels
+There are many labels that people can use to attach to their PR or to issue reports. These are handy for organisation, as sometimes a developer will go through all the PRs that have a certain label (`Size-XXS`, or `A-Chemistry` for instance) and use that to pick what to review. It's therefore good practise to put labels on your pull requests, and there's a couple ways to add labels:
+
+- Keywordlabeler bot assigns labels to pull requests based on certain key words in the pull reuqest's description. For instance, putting \[BUG\] or \[FIX\] in the top line of a PR's ddescription will add the `C-bug` label to it. The full list of what key word matches what label can be found in the .github/keylabeler.yml file. These are handy for adding labels which start with the prefix `A-`.
+- Contributors with [triage permissions](#Triage-Team) can simply add and remove labels from their own PRs, as well as issue reports and other people's PRs.
+
+Some labels are automatically assigned as well, and do certain things:
+
+- Size labels are automatically assigned by github-actions bot, based on the amount of lines changed.
+- Labels like `C-Sprites`, `C-Sound`, and `C-Documentation` are automatically added based on changes to `.dmi` files, `.ogg` files, and `.md` files respectively.
+- The `S-Merge-Conflict` label gets applied when your branch is no longer able to be merged with the master branch. Note that this doesn't update instantly, so the better way to check is to see the line of text at the bottom of the PR that reads "This branch has conflicts that must be resolved", listing the conflicting files. You must resolve merge conflicts, otherwise it can't be merged.
```

Comment:
```suggestion
- The `S-Merge-Conflict` label gets applied when your branch is no longer able to be merged with the master branch. Note that this doesn't update instantly, so the better way to check is to see the line of text at the bottom of the PR that reads "This branch has conflicts that must be resolved", listing the conflicting files. You must resolve merge conflicts, otherwise it can't be merged. For TGUI PRs, conflicts with the `.bundle.js` are less urgent to resolve.
```